src/views/ConsentContinuationOut.vue

```diff
@@ -0,0 +1,822 @@
+<template>
+  <div id="consent-continuation-out">
+    <ConfirmDialog
+      attach="#consent-continuation-out"
+      ref="confirm"
+    />
+
+    <PaymentErrorDialog
+      attach="#consent-continuation-out"
+      filingName="Correction"
+      :dialog="paymentErrorDialog"
+      :errors="saveErrors"
+      :warnings="saveWarnings"
+      @exit="onPaymentErrorDialogExit()"
+    />
```

Comment:
Are dialogs like these common in many components?  Could we simplify the code by listing them in a common place?

Reply:
The local filings (AR, COA, COD, Correction, and now CCO) have dialogs like this.

I'd like to see a clean way to reduce duplication, but this is another pattern I've seen in [Namerequest](https://github.com/bcgov/namerequest/blob/2eedf417b8f8554869fd275c6344b125b1bd091f/src/App.vue#L48) that I don't like (in Namerequest anyway) -- why declare them where they're not needed?

Also, we plan to move AR, COA, COD and Correction to Edit UI ("multi-filigns"), so maybe it's not worth a ton of effort today to refactor those.